src/main/java/ch/uzh/ifi/hase/soprafs22/entity/Game.java

```diff
@@ -46,40 +46,19 @@ public class Game implements Serializable {
   // Getters and Setters
 
   public void setId(long id){this.id=id;}
-
   public Long getId(){return this.id;}
 
-  public void setUserId(long userId){
-      this.userId=userId;
-  }
-
-  public Long getUserId(){
-      return this.userId;
-  }
-
-  public void setBlackCard(BlackCard card){
-        this.blackCard=card;
-  }
-
-  public BlackCard getBlackCard(){
-        return this.blackCard;
-  }
-
-  public void enqueuePlay(Play play){
-    plays.add(play);
-  }
+  public void setUserId(long userId){this.userId=userId;}
+  public Long getUserId(){return this.userId;}
 
-  public List<Play> getPlays() {
-      return this.plays;
-    }
+  public void setBlackCard(BlackCard card){this.blackCard=card;}
+  public BlackCard getBlackCard(){return this.blackCard;}
 
-  public Date getCreationTime() {
-    return creationTime;
-  }
+  public void enqueuePlay(Play play){plays.add(play);}
+  public List<Play> getPlays() {return this.plays;}
 
-  public void setCreationTime(Date creationTime) {
-    this.creationTime = creationTime;
-  }
+  public Date getCreationTime() {return creationTime;}
+  public void setCreationTime(Date creationTime) {this.creationTime = creationTime;}
```

Comment:
Again, if you wish to clean up code do so on a different branch - or is it your IDE which does that?